frontend/src/app/alternatives/AlternativeForm.ts

```diff
@@ -0,0 +1,11 @@
+import { Alternative } from "../Hierarchy";
```

Comment:
Everything looks good, but we should wait for Brad to finish alternatives so that this can be integrated.